source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3628,9 +3745,75 @@ export class AdaptiveCard extends ContainerWithActions {
         return renderedCard;
     }
 
+    updateLayout(processChildren: boolean = true) {
+        super.updateLayout(processChildren);
+        this.handleBottomOverflows();
+    }
+
     canContentBleed(): boolean {
         return true;
     }
+
+    private getChildElements(): Array<CardElement> {
+        var children = [];
+
+        var addChildren = (container: Container) => {
+            for (let item of container.items) {
+                if (item instanceof Container) {
+                    addChildren(item);
+                }
+                else if (item instanceof ColumnSet) {
+                    for (let column of (<ColumnSet>item).columns) {
+                        addChildren(column);
+                    }
+                }
+                else {
+                    children.push(item);
+                }
+            }
+        };
+
+        addChildren(this);
+        return children;
+    }
+
+    private handleBottomOverflows() {
```

Comment:
This needs to be updated so that each element triggers the same logic in its own children instead of relaying on collecting all the elements beforehand. What you have now doesn't work with extensibility.